src/watchOS/bitwarden/bitwarden WatchKit Extension/Services/CryptoService.swift

```diff
@@ -34,7 +34,7 @@ public class CryptoService{
             let item = try JSONDecoder().decode(type, from: decryptedData)
             return item
         } catch {
-            assertionFailure("Fail to decode item for keychain: \(error)")
+            Log.e("Fail to decode item for keychain: \(error)")
```

Comment:
Are we sure this won't log to the console?

Reply:
Yes, it's an internal helper [LoggerHelper](https://github.com/bitwarden/mobile/blob/master/src/watchOS/bitwarden/bitwarden%20WatchKit%20Extension/Helpers/LoggerHelper.swift)
it has a bool indicating that it only logs on DEBUG, and also the swift print func is [wrapped](https://github.com/bitwarden/mobile/blob/master/src/watchOS/bitwarden/bitwarden%20WatchKit%20Extension/Helpers/LoggerHelper.swift#L9) so that it only works on DEBUG to avoid forgetting some print somewhere, so if it happens it will do nothing 😄 😄   (loved that trick)